deploy/images/controller/Dockerfile

```diff
@@ -1,17 +1,23 @@
-# Use distroless image which already includes ca-certificates
-FROM gcr.io/distroless/static:nonroot
+# Use Ubuntu image to enable Bicep CLI.
+# Switch to something more lightweight when we can find a 
+# base image that supports running Bicep. 
+FROM ubuntu:24.04
 
 # Argument for target architecture
 ARG TARGETARCH
 
 # Set the working directory
 WORKDIR /
 
+# Install the required dependencies
+RUN apt-get update && \
```

Comment:
I'm assuming these dependencies are needed to run Bicep?

Reply:
yee